src/cheri/riscv-hybrid-integration.adoc

```diff
@@ -25,19 +25,21 @@ The two execution modes are:
 
 {cheri_int_mode_name}::
 Executing with RV32I/RV64I base ISA.
++
 All {cheri_base_ext_name} instructions and CSRs are also available in {cheri_default_ext_name}.
 +
 IMPORTANT: All instructions from existing RISC-V extensions can be run in {cheri_int_mode_name}.
-  This execution mode is fully binary compatible with a non-CHERI-aware programs.
+  This execution mode is fully binary compatible with non-CHERI-aware programs.
 
 {cheri_cap_mode_name}::
 Executing with the {cheri_base_ext_name} base ISA.
-Some instructions in the RV32I/RV64I base ISA (plus any extensions) use capabilities instead of integers for memory references (see xref:section_existing_riscv_insns[xrefstyle=short]).
++
+Some instructions in the RV32I/RV64I base ISA (and in some other extensions) use capabilities instead of integers for memory references (see xref:section_existing_riscv_insns[xrefstyle=short]). In these cases, such as <<SH1ADD_CHERI>>, the lower XLEN bits of the result is the same in either mode and the upper XLEN bits, and valid tag, are either handled as capabilities in {cheri_cap_mode_name}, or are ignored in the source and written to zero in {cheri_int_mode_name}
```

Comment:
```suggestion
Some instructions in the RV32I/RV64I base ISA (and in some other extensions) use capabilities instead of integers for memory references (see xref:section_existing_riscv_insns[xrefstyle=short]).
In these cases, such as <<SH1ADD_CHERI>>, the lower XLEN bits of the result is the same in either mode and the upper XLEN bits, and valid tag, are either handled as capabilities in {cheri_cap_mode_name}, or are ignored in the source and written to zero in {cheri_int_mode_name}
```

Having one sentence per line makes (future) changes easier to review